_posts/2022-07-06-babelfish-release.md

```diff
@@ -4,14 +4,16 @@ author: Simon Lightstone
 authors: 
     - Simon Lightstone
 comments: true
-title: "Babelfish 2.1.0 has been released with improved support for SSMS, CREATE ROLE support and more"
+title: "Babelfish 2.1.0 has been released (updated)"
 categories:
 - releases
 ---
 
-We’re excited to announce Babelfish 2.1.0, which makes it easier than ever to adopt Babelfish.  Our users have asked us to make Babelfish easier to use, and we’ve responded with a feature-packed release.  After installing 2.1.0, you can browse database tables and column types with SQL Server Management Studio (SSMS).  This release also adds support for creating roles on the TDS port with the CREATE ROLE command, and improves cross-database reference support for queries.
+Update: Babelfish 2.1.0 exposes an issue introduced with community PostgreSQL 14 that is fixed in [PostgreSQL version 14.4](https://www.postgresql.org/docs/release/14.4/).  In response, the Babelfish team has backported the fix and released Babelfish 2.1.0a.  We recommend you move to 2.1.0a as soon as possible to avoid index corruption.
```

Comment:
Shouldn't this be past-tense? (exposed vs exposes)

Reply:
I would argue no, because 2.1.0 still has that problem - if anyone has installed it, they need to update, etc.